templates/report_template.html

```diff
@@ -41,6 +41,7 @@
 					<tr>
 						<th style="text-align: center;">OSP Version </th>
 						<th style="text-align: center;">Job Name</th>
+						<th style="text-align: center;">Build Days ago</th>
```

Comment:
```suggestion
						<th style="text-align: center;">Days since last build</th>
```

Reply:
Could we also swap the positions of "Last Build Number" and the new column such that the new column comes after "Last Build Number"?